src/io.rs

```diff
@@ -14,22 +18,36 @@ where
     F: Fn(&mut I, M) -> Result<M, String>,
 {
     return match File::open(&Path::new(path)) {
-        Ok(mut is) => match ::protobuf::parse_from_reader::<M>(&mut is) {
-            Ok(t) => callback(src, t),
-            Err(_) => Err(String::from("Failed to load file.")),
-        },
+        Ok(mut is) => {
+            match ::protobuf::parse_from_reader::<M>(&mut is) {
+                Ok(t) => callback(src, t),
+                Err(_) => Err(String::from("Failed to load file.")),
+            }
+        }
         Err(_) => Err(String::from("Failed to open file")),
     };
 }
 
+pub fn write_protobuf<I: Interfaceable>(
+    src: &mut I,
+    mut game: Game,
```

Comment:
variable does not need to be mutable
  --> src/io.rs:33:5
   |
33 |     mut game: Game,
   |     ---^^^^^
   |     |
   |     help: remove this `mut`
   |
   = note: #[warn(unused_mut)] on by default